### ISSUE TITLE
Add the missing mouse hover focus indicators to Edit app page

### ISSUE THREAD

**Issue by ctrance** [ucla-oarc-mobile/casa-on-rails#218]
@krozett The following are missing mouse hover focus indicators on Edit app page:
a.	“Save app” button
![image](https://user-images.githubusercontent.com/1034174/108250977-bf1f2380-710b-11eb-9965-6ce53ec4d37a.png)
b.	“privacy policy template generator” link
![image](https://user-images.githubusercontent.com/1034174/108250988-c3e3d780-710b-11eb-9285-266043b3c9e3.png)
c.	“admin > out peers” link
![image](https://user-images.githubusercontent.com/1034174/108251004-c9412200-710b-11eb-8938-4360403a0046.png)
d.	Add author and organization buttons
e.	Remove author and organization buttons
![image](https://user-images.githubusercontent.com/1034174/108251042-d231f380-710b-11eb-95f5-680db2371506.png)
f.	Categories list items
![image](https://user-images.githubusercontent.com/1034174/108251070-d9590180-710b-11eb-80c5-bbf12579eafe.png)
g.	“learning tools interoperability (LTI)” link
![image](https://user-images.githubusercontent.com/1034174/108251099-df4ee280-710b-11eb-95da-5a6ba08762cd.png)
h.	Dropdown lists for LTI consumer, version, and LIS outcomes
![image](https://user-images.githubusercontent.com/1034174/108251122-e544c380-710b-11eb-9304-6568dfc1a5b0.png)
i.	Remove and add LTI configuration buttons
![image](https://user-images.githubusercontent.com/1034174/108251147-ed046800-710b-11eb-95e0-3a5170e75a55.png)

Accessibility success criterion:
* https://www.w3.org/TR/WCAG21/#use-of-color
* https://www.w3.org/TR/WCAG21/#non-text-contrast

**Comment by ctrance** [comment]
Closing old tickets that we will not fix for now.
